repository/c.json

```diff
@@ -3937,6 +3937,19 @@
 				}
 			]
 		},
+		{
+			"author": ["Tomas Barry"],
+			"name": "Create WebP Image",
+			"description": "Convert images to WebP from Sublime Text",
```

Comment:
You can put this in the GitHub repo description instead for ease of maintenance.

Reply:
Addressed.